### DISCUSSION TITLE
revert fix (#3564) in update 3?

### DISCUSSION THREAD

**Discussion by JJKHub** [flybywiresim/aircraft#3635] [General]
Asobo just [posted](https://forums.flightsimulator.com/t/flight-dynamics-bug-details/368499) that the lift change is not permanent:

"lift effect for the flaps (as defined in the flight_model.cfg) is doubled", which was not intentional.

"In the meantime, a workaround is to go into the flight_model.cfg and divide the flap lift by 2x (there are several possible scalars for that purpose: lift_coef_flaps or lift_scalar)."

The quick flight model fix for 1.13.16.0 (#3564), made the a320neo landable again, but will probably break in sim update 3 or in a potential hotfix before that, right?

If this is the case I hope that this discussion may reach one of the fbw coders, so when asobo changes the lift coeficient again,
we can:
1. Detect such an issue very quickly, when Asobos fix drops.
2. Easily revert the quick flight model fix for 1.13.16.0 (#3564) or at least quickly set appropriate values again.

I don't know if Asobos suggested workaround, will break the custom a320 landing behavior of this mod, but if such a similar landing issue is raised by a mod user, the developers might want to ask, if he applied asobos hotfix before pursuing it further, so that no valueable time is wasted unnecessarily.

Is this of help?

Update:
The Hotfix 1.13.17.0 from Asobo adressing the flaps issue ia now available.
#3783 is commited.

Great Job FBW/ @donstim 

The discussion here can now be ignored/deleted.

**Comment by hotshotp** [comment]
I do not think that they will make a hotfix. I will be surprised if asobo makes a hotfix for their issues, they usually make one very late in the updates

**Reply by JJKHub** [reply]
I agree, that was Asobos last stand, but if it is not in a Hotfix that drops very soon, the fix will be included in sim update 3, which should release early March.
